server/methods/core/cartToOrder.js

```diff
@@ -173,6 +173,8 @@ export function copyCartToOrder(cartId) {
   const orderId = Collections.Orders.insert(order);
 
   if (orderId) {
+    Hooks.Events.run("onOrderCreate", order, [orderId]);
```

Comment:
Why would we pass both order and orderId? And why is orderId being passed as the only member of an array?